beacon-chain/beacon-network.md

```diff
@@ -155,7 +155,7 @@ HISTORICAL_ROOTS_LIMIT = 2**24  # = 16,777,216
 
 ```
 light_client_bootstrap_key = Container(block_hash: Bytes32)
-selector                   = 0x00
+selector                   = 0x03
```

Comment:
Updating this to `0x10` (and `0x11`, `0x12`, etc) would give us a nice namespace and give us room to expand the history network set of keys without awkward gaps.